docs/dev/pypi_publish/pypi-openvino-dev.md

```diff
@@ -165,10 +165,6 @@ To avoid such issues you can also disable globbing for PIP commands by defining
 alias pip='noglob pip'
 ```
 
-### ERROR:root:Could not find the Inference Engine or nGraph Python API.
-
-On Windows*, some libraries are necessary to run OpenVINO. To resolve this issue, install the [C++ redistributable (.exe)](https://aka.ms/vs/17/release/vc_redist.x64.exe). You can also view a full download list on the [official support page](https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist).
```

Comment:
I think we should restore this line with title "Could not find the OpenVINO Python API."